draft-ietf-quic-recovery.md

```diff
@@ -508,6 +508,12 @@ until it is certain that the server has finished its address validation
 probe timer if the client has not received an acknowledgement for one of its
 Handshake or 1-RTT packets.
 
+When a server receives duplicate CRYPTO data in an Initial packet after sending
+its Initial flight, it can assume the client did not receive all Initial CRYPTO
+data. To speed handshake completion, it SHOULD retransmit all unacknowledged
+Initial CRYPTO data subject to the path validation limits.  After doing so,
+the PTO is re-armed.
```

Comment:
The server can't arm the PTO prior to confirming that the path is valid.

Reply:
The server can't send if it's limited by the amplification factor, but the PTO can be armed otherwise.

Reply:
The way I read this, it says that you retransmit then you re-arm the PTO timer.

I think that you mean to say that when the path is validated, you can re-arm the PTO timer after retransmitting the CRYPTO data, but that's not what it says.

Reply:
The PTO can be armed prior to path validation, but the text said it always was, which is wrong, so I reworded the paragraph.